message_validator.py

```diff
@@ -18,7 +18,7 @@ def validate_message(request, username, is_inbound):
     if not 'text' in request.json:
         response =  Response('','text is missing')
         return response
-    if len(request.json['text']) < 1 or len(request.json['text']) > 120:
+    if len(request.json['text']) < 1 or len(request.json['text']) > 150:
         response =  Response('','text is invalid')
```

Comment:
**Suggestion:** The Response object is being created with an empty string as the first argument. It would be more informative to provide a meaningful message.
```suggestion
    if not 'text' in request.json:
        response =  Response('text is missing', status=400)
        return response
    if len(request.json['text']) < 1 or len(request.json['text']) > 150:
        response =  Response('text is invalid', status=400)
        return response
```